js/plugins/vertexai/src/index.ts

```diff
@@ -86,6 +142,8 @@ export interface PluginOptions {
     metrics: VertexAIEvaluationMetric[];
   };
   modelGardenModels?: ModelReference<any>[];
+  projectNumber?: string;
```

Comment:
Hmm... can you use the `projectId` which we automatically get from the GoogleAuth?

Reply:
yup for sure, prototyping was easier to just manually add it but I'll automate this